src/evaluation/model_results_export.py

```
@@ -22,273 +22,276 @@
         """
         logger.info("Loading raw 65% M+1 prices from continuous futures...")
-        
+
         # Load continuous futures data (contains both raw and adjusted prices)
         continuous_path = Path(self.config["data"]["continuous_futures"])
         df = pd.read_pickle(continuous_path)
-        
+
         # Extract raw price and Y target from consolidated features for alignment
         consolidated_path = Path(self.config["data"]["consolidated_features"])
         consolidated_df = pd.read_pickle(consolidated_path)
```

Ruff (0.12.2)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)